jellyfishplugin.py

```diff
@@ -98,8 +102,8 @@ def off(self) -> bool:
         Returns:
             True if device seems to have been turned off.
         """
-        print('Turning off "%s" for zones ["%s"]' % (self.name, self.zones))
-        cmd = '{"cmd":"toCtlrSet","runPattern":{"file":"","data":"","id":"","state":0,"zoneName":["%s"]}}' % self.zones
+        print('Turning off "%s" with pattern "%s" for zones ["%s"]' % (self.name, self.pattern, self.zones))
+        cmd = '{"cmd":"toCtlrSet","runPattern":{"file":"%s","data":"","id":"","state":0,"zoneName":["%s"]}}' % (self.pattern, self.zones)
```

Comment:
Setting the pattern for `off` seems odd but I tested today and it does work--the pattern is changed and lights are turned off.

If lights are **on** for `Pattern A` and turned **off** with `Pattern B` turning them **on** again without specifying a pattern will start them with `Pattern B`.